protobuf/abci.proto

```diff
@@ -207,13 +207,13 @@ message ResponseCommit {
 // ConsensusParams contains all consensus-relevant parameters
 // that can be adjusted by the abci app
 message ConsensusParams {
-  BlockSizeParams block_size = 1;
+  BlockParams block = 1;
   EvidenceParams evidence = 2;
   ValidatorParams validator = 3;
 }
 
-// BlockSize contains limits on the block size.
-message BlockSizeParams {
+// BlockParams contains limits on the block size and timestamp.
+message BlockParams {
```

Comment:
The comment mentions the timestamp but the BlockParams msg doesn't contain anything about the timestamp.

Reply:
this file is from https://github.com/tendermint/tendermint/blob/master/abci/types/types.proto#L215 changed with https://github.com/tendermint/rust-abci/blob/master/Makefile#L14

Reply:
Makes sense. Thanks! We probably forgot to update the comment there...